launcher/Application.cpp

```diff
@@ -249,7 +250,8 @@ Application::Application(int& argc, char** argv) : QApplication(argc, argv)
     parser.process(arguments());
 
     m_instanceIdToLaunch = parser.value("launch");
-    m_serverToJoin = parser.value("server");
+    m_server_to_join = parser.value("server");
+    m_world_to_join = parser.value("world");
```

Comment:
but... everything else is camelCase in the launcher codebase... if anything I'd get rid of the `m_` prefix...

Reply:
If I requested using snake case it is because snake case was being used in other places in the file. The `m_` prefix is usually used to avoid conflicts with function names.